locales/es.po

```diff
@@ -1281,101 +1313,101 @@ msgstr "Espectador"
 msgid "Start Server"
-msgstr ""
+msgstr "Iniciar servidor"
 
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:7:18
 msgid "Started"
-msgstr ""
+msgstr "Iniciado"
 
 #: src/renderer/pages/home/sidebar/ranked_status.messages.ts:6:21
 msgid "Starting In"
-msgstr ""
+msgstr "Comienza en"
 
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:9:25
 msgid "Starting Server..."
-msgstr ""
+msgstr "Iniciando servidor..."
 
 #: src/renderer/pages/home/sidebar/ranked_status.messages.ts:4:23
 msgid "Starting Soon"
-msgstr ""
+msgstr "Comienza pronto"
 
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:8:19
 msgid "Starting..."
-msgstr ""
+msgstr "Iniciando..."
 
 #: src/renderer/pages/spectate/share_gameplay_block/broadcast_panel.messages.ts:2:17
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:4:17
 msgid "Status"
-msgstr ""
+msgstr "Estado"
 
 #: src/renderer/pages/spectate/share_gameplay_block/broadcast_panel.messages.ts:7:24
 msgid "Stop Broadcast"
-msgstr ""
+msgstr "Detener transmisión"
 
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:6:21
 msgid "Stop Server"
-msgstr ""
+msgstr "Detener servidor"
 
 #: src/renderer/pages/spectate/spectate_remote_control_block/spectate_remote_control_block.messages.ts:10:18
 msgid "Stopped"
-msgstr ""
+msgstr "Detenido"
 
 #: src/renderer/pages/console_mirror/add_connection_dialog/add_connection_form.messages.ts:30:17
 msgid "Submit"
-msgstr ""
+msgstr "Enviar"
 
 #: src/renderer/pages/settings/chat_settings/chat_settings.messages.ts:5:20
 msgid "Subscribe"
-msgstr ""
+msgstr "Suscribirse"
 
 #: src/renderer/pages/settings/advanced_app_settings/advanced_app_settings.messages.ts:8:28
 msgid "Successfully cleared temporary files."
 msgstr "Archivos temporales eliminados correctamente."
 
 #: src/renderer/pages/settings/help_page/support_box/support_box.messages.ts:10:29
 msgid "Successfully copied logs to clipboard"
-msgstr ""
+msgstr "Registros copiados exitosamente al portapapeles"
 
 #: src/renderer/components/footer/footer.messages.ts:2:24
 msgid "Support Slippi"
-msgstr ""
+msgstr "Apoyar a Slippi"
 
 #: src/renderer/pages/settings/help_page/support_box/network_diagnostics_button/network_diagnostic_result/network_diagnostics_result.messages.ts:15:23
 msgid "Symmetric NAT"
-msgstr ""
+msgstr "NAT simétrico"
 
 #: src/renderer/pages/console_mirror/add_connection_dialog/add_connection_form.messages.ts:3:23
 #: src/renderer/pages/console_mirror/saved_connections_list/saved_connection_item.messages.ts:12:23
 msgid "Target Folder"
-msgstr ""
+msgstr "Carpeta de destino"
 
 #: src/renderer/pages/replays/replay_file_stats/game_profile_header/game_profile_header.messages.ts:4:21
 msgid "Target Test"
-msgstr ""
+msgstr "Target Test"
```

Comment:
- This is.... a hard one. Bookmarked!

Reply:
i think it was suggested to just not translate "Target Test" and "Home-Run Contest" but up to you! i don't know what the pal version says